### ISSUE TITLE
Node.js: Need to setup js/window

### ISSUE THREAD

**Issue by mfikes** [ScalaConsultants/replumb#21]
If using Replumb in Node.js, I get this error at runtime:

```
Initializing REPL environment...
Exception caught in read-eval-call:  #object[ReferenceError ReferenceError: window is not defined]
Calling back!
 {:opts {:verbose true},
 :data {},
 :res
 {:error #object[ReferenceError ReferenceError: window is not defined]}}

{:success? false, :form nil, :error #object[ReferenceError ReferenceError: window is not defined]}
```

I found that I can work around this via having the following in my client code:

```clojure
(js* "var window = global;")
```

**Comment by arichiardi** [comment]
This one is due to the fact that I am trying to write properties on `window` that in `node` is not present. I won't be necessary anymore with the new patch.

**Comment by arichiardi** [comment]
Closed by #30.
